packages/trpc/server/routers/viewer/bookings/_router.tsx

```diff
@@ -75,7 +75,7 @@ export const bookingsRouter = router({
     });
   }),
 
-  confirm: bookingsProcedure.input(ZConfirmInputSchema).mutation(async ({ input, ctx }) => {
+  confirm: authedProcedure.input(ZConfirmInputSchema).mutation(async ({ input, ctx }) => {
```

Comment:
Hey @SomayChauhan, if we use the booking procedure here, then we always get an unauthorized error if the current user is not the organizer of the booking. Therefore, it would not be possible for the admin/owner to confirm or reject the booking.

In fact, the booking procedure is nothing else but extracting booking details based on some checks and updating the context on top of the auth procedure. I don't think it would be a good idea to update or change the booking procedure for this feature, as it is used for other handlers as well.

So instead, we should use the auth procedure and perform a similar check inside the confirm handler. Let me know what you think?